<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThemLopTinChi</class>
 <widget class="QDialog" name="ThemLopTinChi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>571</width>
     <height>591</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Mã Môn Học:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="txtMaMH"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Niên Khóa:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="txtNienKhoa">
      <property name="inputMask">
       <string>0000-0000;_</string>
      </property>
      <property name="placeholderText">
       <string>VD: 2024-2025</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Học Kỳ:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="spinHocKy">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nhóm:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QSpinBox" name="spinNhom">
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Số sinh viên tối thiểu:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QSpinBox" name="spinMin">
      <property name="minimum">
       <number>15</number>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
      <property name="singleStep">
       <number>5</number>
      </property>
      <property name="value">
       <number>20</number>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Số sinh viên tối đa:</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QSpinBox" name="spinMax">
      <property name="minimum">
       <number>15</number>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
      <property name="singleStep">
       <number>5</number>
      </property>
      <property name="value">
       <number>20</number>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Hủy Lớp:</string>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QCheckBox" name="checkHuy">
      <property name="text">
       <string>Lớp bị hủy</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>610</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnLuu">
      <property name="toolTip">
       <string>Lưu lớp tín chỉ</string>
      </property>
      <property name="text">
       <string>Lưu</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnHuy">
      <property name="toolTip">
       <string>Đóng cửa sổ mà không lưu</string>
      </property>
      <property name="text">
       <string>Hủy</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
